### ISSUE TITLE
Add upcoming generation type: Aerial Wind

### ISSUE THREAD

**Issue by toddmedema** [toddmedema/electrify#124]
https://cleantechnica.com/2024/07/22/follow-the-money-kitemill-aims-for-wind-energy-with-90-less-infrastructure/

**Comment by toddmedema** [comment]
## Implementation research: Airborne Wind Energy (AWE)

Research current as of **2026-08-29**. The short version: this is implementable as a deliberately forward-looking generator, but it should be called **Airborne Wind** (the term used by the industry, IEA Wind, IEC work, and the EU projects) rather than “Aerial Wind.” It should model a **rigid-wing, ground-generation pumping system**—a kite repeatedly reels a tether out through a ground generator and consumes some energy while reeling back in. That is the Kitemill design and is specific enough to produce defensible game parameters.

### Recommendation

Add it with a conservative **2030 unlock**, a distinct high-altitude wind curve, and explicit early-commercial economics. Do not make it a reskinned onshore turbine: its cut-in/rated behavior, altitude, unit scale, maintenance mix, and regulatory constraints differ materially.

| Game field | Recommended first implementation | Confidence / basis |
|---|---:|---|
| `name` / `fuel` | `"Airborne Wind"` | High; standard term is Airborne Wind Energy (AWE) |
| `description` | `Higher, steadier winds with light infrastructure, but immature and maintenance-heavy` | High-level tradeoff supported by IEA/EC evidence |
| `available` | `year >= 2030` | High; Kitemill's revised pilot schedule reaches mature operation in 2030 |
| Reference unit | 100 kW | High; Kitemill KM2 and NAWEP |
| Planned first grid-connected array | 1.2 MW (12 × 100 kW) | High; revised 2026 NAWEP application |
| `maxPeakW` | Anchor at 1.2 MW in 2028, then grow with a capped learning rule | Anchor high-confidence; growth is gameplay extrapolation (proposal below) |
| `buildCost` | About **$7.0/W in 2028**, trending to an optimistic **$4.1/W by 2035**, then flat | Medium/low; derivation below, with large uncertainty |
| `annualOperatingCost` | **$51.4/kW-year** (`0.0514 * peakW`) | Medium; Kitemill's €45.5/kW-year series-production estimate × ECB 2025 average $1.13/€ |
| `capacityFactor` | Weather-derived; validate/calibrate to about **40% at Lista** | High target, medium transferability; 3,500 full-load hours/year |
| Power curve at 100 m reference height | 6 m/s cut-in, 11 m/s rated, 20 m/s cut-out | Medium/high; peer-reviewed 500 kW fixed-wing ground-gen optimum |
| `yearsToBuild` | `2 + magnitude / 3` | Medium; revised NAWEP installation is phased over 2026–2028 |
| `spinMinutes` | 1 | High; weather-driven/must-run renewable in this simulation |
| `btuPerWh` | 0 | High |
| `lifespanYears` | 25 | Medium; reference economic/scaling model assumption |
| `annualOutputDegradation` | Omit initially | High; no defensible fleet-age evidence, and tether/kite replacement belongs in O&M |

### Commercial-readiness check

The original CleanTechnica article repeats useful company claims, but it predates stronger evidence and should not be the numerical source.

* The EU's current AWE-KM2 record says KM2 is a **100 kW first commercial system under development**. Its first reporting period (through September 2025) still describes validation and automated-landing work on the smaller KM1, with commercialization of KM2 as the goal. The project runs through September 2026.
* Kitemill received Norwegian BVLOS approval in October 2025, an important operational milestone, but in early 2026 it still described endurance, power optimization, and fully automated end-to-end operation as upcoming work. It flew 254 intentionally short KM1 test flights in 2025.
* The revised April 2026 NAWEP license application is the best current deployment source. It proposes 12 production KM2 units plus 3 R&D units at Lista Airport, 1.2 MW of grid feed-in, installation/grid work through 2028, initial operation in 2029, and mature operation in 2030. This is explicitly an R&D/demonstration plant.
* The schedule has already slipped materially: the EU's original Innovation Fund factsheet listed July 2024 as entry into operation, while the current plan reaches commissioning in 2028 and mature production in 2030. That argues against unlocking in 2024–2027 merely because prototypes exist.

Therefore, `year >= 2030` is the cleanest game rule: it represents the first documented mature-operation target rather than prototype availability. If the game wants the player to participate in the pilot phase, 2028 is the earliest defensible alternative, but it should have a much lower initial utilization (the application forecasts only 1,200 full-load hours in 2028 and 2,000 in 2029, versus 3,500 from 2030).

### Output and weather model

The NAWEP specification gives an operating envelope of **5–25 m/s** and **150–500 m above ground**. Its mature estimate is **4.2 GWh/year from 1.2 MW**, exactly 3,500 full-load hours, or a **39.95% capacity factor**. Kitemill says current KM1 trajectories average about 220 m.

For a generic fixed-wing ground-generation facility, the best complete public power-curve anchor is the peer-reviewed 2025 TU Delft design study. Its LCOE-optimal 500 kW system uses a 100 m reference wind speed and has **6 m/s cut-in, 11 m/s rated, and 20 m/s cut-out**. A separate 2024 paper explains that ground-generation systems have a low-wind cubic region, then a tether-force-limited linear region, then rated output. The exact knee is design-specific, so v1 can use the game's existing linear interpolation between cut-in and rated; this is more transparent than inventing a hidden knee.

Preferred implementation:

1. Add `getAirborneWindOutputFactor()` and `getAirborneWindCapacityFactor()` rather than changing the existing onshore curve.
2. Use wind at **100 m reference height**. Best fidelity is to add Open-Meteo/ERA5 `wind_speed_100m` to the weather fetch/binary format. IEA Task 48 finds that optimal operating heights are generally below 400 m and often around 200 m, but the peer-reviewed curve already expresses its inputs at 100 m while internally accounting for the operating trajectory; feeding it “500 m wind” would double-count the height benefit.
3. If refetching every weather file is out of scope, derive a provisional 100 m value from the existing raw 10 m wind with a power-law shear exponent of **0.20**, the reference study's value: `v100 = v10 * (100 / 10)^0.20`. This must start from the raw weather value, not an already-adjusted value.
4. Apply one explicit availability/loss factor only after running the historical Lista coordinates through the model; calibrate that factor so the long-run modeled CF is approximately 0.40. Then freeze it and report CF by game location from the local weather, rather than hard-coding 40% everywhere.

Important existing-code trap: `generateNewTimeline()` already multiplies `WIND_KPH` by `OUTSKIRTS_WIND_MULTIPLIER`, and `getWindOutputFactor()` applies that multiplier again before its legacy `/ 5` derate. Reusing that function or feeding its adjusted value into a new physical wind-shear formula would silently double-scale the resource. A direct 100 m field avoids this ambiguity; the fallback should explicitly undo/bypass the timeline multiplier.

The result should remain intermittent and must-run. It should be dispatched like Sun/Wind/Offshore Wind and should automatically land at low wind or severe-weather cut-out, which the zero-output portions of the curve represent.

### Economics

There is no observed commercial cost series yet, so the game should expose uncertainty rather than adopting Kitemill's promotional “50% lower LCOE” claim as a present fact.

**FOAK/pilot anchor.** The original EU Innovation Fund record lists €7.445 million of relevant cost for a 1.2 MW project: **€6.20/W**. At the ECB's 2025 annual average of $1.13/€, that is approximately **$7.00/W**. The newer license application independently estimates 76–85 MNOK total pilot CAPEX, including 15 systems (3 R&D), grid works, SCADA, engineering, commissioning, and contingency; this confirms the same order of magnitude but cannot be cleanly divided by the 1.2 MW revenue capacity.

**Early-series floor.** The peer-reviewed 2025 reference design finds €142/MWh for an LCOE-optimal 500 kW system. It reports €2.6 million of 25-year lifetime OpEx and an approximately 0.7:1 CapEx-to-lifetime-OpEx ratio, implying about €1.82 million CapEx, or **€3.64/W** (**$4.11/W** at the same exchange rate). This is an inference, not an observed price, and the authors explicitly say their early-commercial model assumes 50+ units and excludes development overhead, manufacturing overhead, and profit margins. Treat it as an optimistic floor, not the launch price.

A reasonable bounded curve is therefore:

```ts
function airborneWindCostPerW(year: number): number {
  return costBetween(year, 2028, 7.0, 2035, 4.1);
}
```

Use `scaledBuildCost(airborneWindCostPerW(year), 1_200_000, peakW)` so the source-sized 1.2 MW array lands on the published all-in cost while retaining the game's established 25%-fixed/75%-variable economies-of-scale convention. Hold the curve flat after 2035 rather than allowing speculative exponential learning to drive it toward zero.

**O&M.** The revised NAWEP application gives a much better implementation input than older conceptual studies: its KM2 series-production LCOE model uses **€45.5/kW-year** baseline O&M. Converted with the 2025 ECB average, this is **$51.4/kW-year**, so:

```ts
annualOperatingCost: 0.0514 * peakW
```

Do not use the pilot's 3.5–5.0 MNOK/year total: it includes R&D personnel, bird campaigns, insurance, grid charges, ground rent, and administration specific to the demonstration. The relatively high recurring cost versus onshore wind is credible: the academic model identifies tether, power-smoothing storage, and component replacements as important lifecycle costs. Those replacements are why a low material bill does not automatically mean near-zero O&M.

### Scale and build time

The UI's plant should represent an **array of standardized units**, not one huge kite. Both the current commercial target (100 kW) and the 2025 scaling study argue against simply scaling a wing to turbine-like multi-megawatt size: modeled LCOE is best around 500 kW per system and larger kites suffer a mass penalty. Arrays can grow by adding units.

For `maxPeakW`, the only deployed-project anchor is 1.2 MW. One possible conservative gameplay extrapolation is:

```ts
maxPeakW: Math.min(
  500_000_000,
  1_200_000 * Math.pow(2, (year - 2028) / 2),
)
```

The 1.2 MW anchor is sourced; the two-year doubling and 500 MW cap are explicitly game balancing assumptions and should be tested. The revised application also describes a proposed factory reaching 176 MW/year at maturity in 2035 and about 1.136 GW cumulative installations during 2027–2038, so arrays reaching tens of MW in the 2030s are at least consistent with the developer's industrial plan, though not yet demonstrated.

Use about two years for a first 1 MW-class array. Although the hardware is modular and avoids large concrete foundations and new roads, NAWEP's real deployment includes airspace, environmental, grid, and commissioning work over 2026–2028. The actual ground equipment is reversible and relocatable, but permitting is part of bringing a power plant online. `2 + magnitude / 3` follows the codebase's size-scaling convention without pretending installation is instantaneous.

### Location, safety, and environmental scope

AWE is not physically location-free. It needs segregated/managed airspace, a ground-risk buffer, grid access, and acceptable bird/noise impacts. IEA Task 48 says commercial operations are expected to be authorized as unmanned aircraft in the EASA “specific” category using risk assessment, and AWE-specific IEC 61400 standards are still under development. Kitemill's revised application operates at an airport inside a designated danger area and uses phased bird monitoring and operating restrictions.

For v1, I would **not invent finite site counts** for every game city; no comparable global, project-ready dataset exists in the repo. Treat each city as a regional utility with a suitable permitted site somewhere in its territory, just as the game abstracts many other siting decisions. A later fidelity pass could add `resources.airborneWind` or a site inventory based on airspace/protected-area screening. The facility description and longer lead time should carry the regulatory downside for now.

Likewise, the 80–90% material reduction, low permanent land take, and lower lifecycle emissions are genuine advantages but currently do not map to a player-visible game stat. They should inform copy, not distort build cost or output. IEA Task 48's independent LCA summary is more conservative than the company's “90%” headline: one fixed-wing configuration saved up to 70% of material and about 40% of GHG potential.

### Code integration checklist

This addition crosses more than `Facilities.tsx`:

* `src/Types.tsx`: add `"Airborne Wind"` to `FuelNameType` and `FuelProductionType`.
* `src/helpers/Energy.tsx`: add the output/capacity-factor helpers and include it in `MUST_RUN_FUELS`.
* `src/data/Facilities.tsx`: calculate the airborne CF from forecast wind and add the generator object/cost curve.
* `src/reducers/Game.tsx`: add a supply switch case using the new factor. Keep display CF and tick output on the same helper.
* Weather path (preferred): add a 100 m wind field through `fetch-weather.js`, `WeatherBinary.tsx`, `RawWeatherType`, forecast climatology/random streams, and `TickPresentFutureType`; bump/handle the binary version. The lower-scope fallback is a documented shear conversion from raw 10 m wind.
* `src/components/base/GameAppBar.tsx`: include it in `WEATHER_DRIVEN_FUELS`.
* `src/Theme.tsx`: assign accessible light/dark chart colors distinct from Wind and Offshore Wind.
* `public/images/airborne wind.svg`: required because the build view resolves `/images/${generator.name.toLowerCase()}.svg`.
* Tests: power-curve boundaries; Lista calibration target; availability year; 2028/2035 costs; O&M; max-size growth; must-run dispatch ordering; supply switch; chart/theme coverage; save/replay round-trip.
* Custom-game caveat: its generator-size list starts at 100 MW and filters out technologies whose `maxPeakW` is smaller. A source-faithful early AWE cap will therefore be absent from the custom-game technology picker unless 1/10 MW sizes are added or the picker is decoupled from its starting-fleet size list. The ordinary build slider already starts at 1 MW.

### Acceptance criteria I would use

1. Locked through 2029; available from 2030.
2. In constant 2025 USD before the game's inflation multiplier, a 1.2 MW source-sized build is approximately $8.4 million at the proposed 2028 FOAK point and $4.9 million at the 2035 early-series point.
3. Annual O&M is about $61,700 for 1.2 MW.
4. Output is zero below 6 m/s and above 20 m/s at the 100 m reference, reaches nameplate at 11 m/s, and never exceeds nameplate.
5. A long-run Lista calibration run is near 40% CF; other cities vary from their own wind records.
6. Airborne Wind appears before dispatchable generators in the supply stack and does not consume fuel or emit operational CO2.
7. Existing saves and historical wind/offshore-wind output remain bit-for-bit unchanged.

### Primary / technical sources

* [Kitemill revised NAWEP concession application, April 2026](https://kitemillwebstorage.blob.core.windows.net/public-vid/nawep-files/2026-04/nawep-concession-web-2026-04.html) — current deployment schedule, 100 kW units, 1.2 MW grid capacity, 4.2 GWh/year, 150–500 m altitude, 5–25 m/s operation, pilot CAPEX, and series O&M.
* [EU CORDIS AWE-KM2 project and first periodic report](https://cordis.europa.eu/project/id/101189207/reporting) — KM2 development status through September 2025 and project end date.
* [Kitemill's January 2026 operational update](https://www.kitemill.com/news/expanding-the-operational-envelope-on-a-foundation-of-efficiency) and [October 2025 BVLOS approval](https://www.kitemill.com/news/bvlos-approval-marks-major-step-for-kitemill) — current test cadence and remaining endurance/automation work (developer sources, so treated as status rather than independent validation).
* [EU Innovation Fund NAWEP factsheet](https://climate.ec.europa.eu/system/files/2022-07/if_pf_2021_nawep_en.pdf) — original €7.445M relevant cost and 1.2 MW scope (also documents the superseded 2024 schedule).
* [IEA Wind Task 48 Phase I final technical report](https://iea-wind.org/wp-content/uploads/2025/12/Task-48-Final-Technical-Report-Phase-I_2025-11-03.pdf) — sector maturity, operating heights, safety/standards, and independent LCA summary.
* [Joshi, von Terzi & Schmehl (2025), system design and scaling trends](https://wes.copernicus.org/articles/10/695/2025/) — 500 kW optimum, power-curve anchors, 25-year economics, LCOE and scaling limits.
* [Joshi, Schmehl & Kruijff (2024), power curve modelling](https://wes.copernicus.org/articles/9/2195/2024/) — pumping-cycle behavior and cubic/linear/rated regimes.
* [IEA Wind Task 48 reference economic model (2024)](https://repository.tudelft.nl/record/uuid%3A3d6b8776-9c40-4f02-b1b8-03cccf14e838) — component-level CapEx/OpEx framework and limits of the current estimates.
* [Open-Meteo Historical Weather API](https://open-meteo.com/en/docs/historical-weather-api) — confirms that the same archive endpoint already used by the repository exposes hourly `wind_speed_100m` back to 1940.
* [ECB 2025 annual average USD/EUR reference rate](https://data.ecb.europa.eu/data/datasets/EXR/EXR.A.USD.EUR.SP00.A) — 1.1300 USD per EUR used only for the game-dollar conversions above.

The key product decision is whether to ship the low-scope 10 m-to-100 m approximation or extend the weather format with 100 m wind. Everything else now has a defensible implementation value; the cost learning curve and array growth rate should remain clearly labeled balance assumptions until commercial data exists.
